admin/acceptance_tests/layers_wmts_test.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=no-self-use
+
+import pytest
+
+from . import check_grid_headers
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures("dbsession")
+def insertLayerWMTSTestData(dbsession):
+    from c2cgeoportal_commons.models.main import \
+        LayerWMTS, RestrictionArea
+
+    dbsession.begin_nested()
+    restrictionareas = []
+    for i in range(0, 5):
+        restrictionarea = RestrictionArea(
+            name="restrictionarea_{}".format(i))
+        dbsession.add(restrictionarea)
+        restrictionareas.append(restrictionarea)
+
+    for i in range(0, 25):
+        name = 'layer_wmts_{}'.format(i)
+        layer = LayerWMTS(name=name)
+        layer.layer = name
+        layer.url = 'https://server{}.net/wmts'.format(i)
+        layer.restrictionareas = [restrictionareas[i % 5],
+                                  restrictionareas[(i + 2) % 5]]
+        dbsession.add(layer)
+    yield
```

Comment:
It also possible to keep data as in
https://github.com/camptocamp/c2cgeoportal/pull/3272/files#diff-376f71eaa343effe987e81143ae636b1R68
For further 'expected' building so not to need to requery.

Reply:
I understand that we should yield the instances : `yield test_data`